sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -38,12 +34,13 @@ class ACRExchangeClient(object):
     :param credential: Credential which provides tokens to authenticate requests
     :type credential: ~azure.core.credentials.TokenCredential
     :keyword api_version: API Version. The default value is "2021-07-01". Note that overriding this default value
-     may result in unsupported behavior.
+        may result in unsupported behavior.
     :paramtype api_version: str
+    :keyword credential_scopes: The scopes that access token can request.
+    :paramtype credential_scopes: str
```

Comment:
```suggestion
    :paramtype credential_scopes: List[str]
```

Reply:
Why it's a list? The `credential_scopes` is paasing to `get_token()` call in function `exchange_add_token_for_refresh_toekn()`, while the expected expect `scpopes` type is `str`: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/credentials.py#L27